packages/eslint-plugin/src/rules/no-confusing-void-expression.ts

```diff
@@ -112,6 +120,15 @@ export default createRule<Options, MessageId>({
         if (invalidAncestor.type === AST_NODE_TYPES.ArrowFunctionExpression) {
           // handle arrow function shorthand
 
+          if (options.ignoreVoidInVoid) {
+            if (
+              invalidAncestor.returnType?.typeAnnotation.type ===
+              AST_NODE_TYPES.TSVoidKeyword
+            ) {
+              return;
+            }
+          }
+
```

Comment:
```suggestion
          if (options.ignoreVoidInVoid && invalidAncestor.returnType?.typeAnnotation.type === AST_NODE_TYPES.TSVoidKeyword) {
            return;
          }

```

[Style] Totally non-blocking, I just noticed that we can inline these two conditions. Feel free to ignore this, if you find the current implementation more readable.

Reply:
thank you i agree this style